packages/cli/src/commands/rewards/show.ts

```diff
@@ -0,0 +1,259 @@
+import { Validator, ValidatorGroup } from '@celo/contractkit/lib/wrappers/Validators'
+import { flags } from '@oclif/command'
+import BigNumber from 'bignumber.js'
+import { cli } from 'cli-ux'
+import { EventLog } from 'web3/types'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { Flags } from '../../utils/command'
+
+export default class Show extends BaseCommand {
+  static description =
+    'Show rewards information about a voter, registered Validator, or Validator Group'
+
+  static flags = {
+    ...BaseCommand.flags,
+    address: Flags.address({ required: false, description: 'Address to filter' }),
```

Comment:
No need for `required: false`
```suggestion
    address: Flags.address({ description: 'Address to filter' }),
```